api/pytest.ini

```diff
@@ -5,5 +5,21 @@ pythonpath = .
 
 filterwarnings=
   ignore:Unverified HTTPS request
+# Ignore warnings related to unverified HTTPS requests.
+# Reason: This warning is suppressed to avoid raising warnings when making HTTP requests
+# to servers with invalid or self-signed SSL certificates. It allows the tests to proceed
+# without being interrupted by these warnings.
   ignore:DateTimeField AccessToken\.expires received a naive datetime
+# Ignore warnings related to naive datetimes in 'DateTimeField' and 'AccessToken.expires' fields.
```

Comment:
I couldn't find much history for why this was added, but I investigated by turning the suppression off and checking to see where the warning is thrown. It looks like we have timezone issues stemming from our use of the AccessTokenFactory [here](https://github.com/WordPress/openverse/blob/main/api/test/factory/models/oauth2.py#L50) using `faker`.

I think we might be able to fix this and opened #2497 to do so, but in any case: the reason this warning was suppressed was because it was only an issue with our factory generated test data.

Reply:
#2497 was merged, so you can rebase this branch on `main` and this warning should no longer be suppressed -- this comment can be removed 🎉